src/popup/router/components/AssetSelector.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <select
+    class="asset-selector"
+    :value="JSON.stringify(currentToken)"
+    @change="(event) => handleChange(event.target.value) "
```

Comment:
"handleChange" => in function use const { value } = event.target

Reply:
Same here - not my code.